docs/source/notebooks/power_analysis.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # %config InlineBackend.figure_format = 'svg'
</pre><p>uncomment and set this to 'retina' (not 'svg') for high resolution graphical output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/277/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done!